R/available-updates.R

```diff
@@ -158,9 +158,9 @@ gitlabUpdates <- function(lib.loc = .libPaths()) {
     do.call(rbind, enumerate(DESCRIPTIONS, function(x) {
       url <- file.path("https://gitlab.com/",
                        "api/v4/projects/",
-                       paste0(x[, "RemoteUsername"],
-                              "%2F",
-                              x[, "RemoteRepo"]),
+                       URLencode(paste0(x[, "RemoteUsername"],
```

Comment:
CC @kevinushey - we don't see any callers of the internal `available_updates` function; do you know of uses that make this code live?